cms/djangoapps/contentstore/rest_api/v1/serializers/textbooks.py

```diff
@@ -0,0 +1,32 @@
+"""
+API Serializers for textbooks page
+"""
+
+from rest_framework import serializers
+
+
+class CourseTextbookChapterSerializer(serializers.Serializer):
+    """
+    Serializer for representing textbook chapter.
+    """
+
+    title = serializers.CharField()
+    url = serializers.CharField()
+
+
+class CourseTextbookItemSerializer(serializers.Serializer):
+    """
+    Serializer for representing textbook item.
+    """
+
+    id = serializers.CharField()
+    chapters = CourseTextbookChapterSerializer(many=True)
+    tab_title = serializers.CharField()
+
+
+class CourseTextbooksSerializer(serializers.Serializer):
+    """
+    Serializer for representing course's textbooks.
+    """
+
+    textbooks = serializers.ListField()
```

Comment:
I guess it should be
```suggestion
    textbooks = CourseTextbookItemSerializer(many=True)
```
right?

Reply:
Maybe this was intentional, idk